backend/database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,12 +14,19 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->id();
-            $table->string('name');
+            $table->uuid('id')->primary();
+            $table->string('user_name');
+            $table->string('password');
+            $table->boolean('is_admin')->default(false);
+            $table->string('first_name');
+            $table->string('last_name');
+            $table->string('middle_name')->nullable();
             $table->string('email')->unique();
+            $table->string('address');
+            $table->dateTime('birthday');
             $table->timestamp('email_verified_at')->nullable();
```

Comment:
ah i think we can remove this. we will not have a verify email feature rigth? we can add it in the future if ever